### ISSUE TITLE
I can't activate Blokada

### ISSUE THREAD

**Issue by Jaheira** [blokadaorg/blokada#347]
Hello,

I can't start Blokata. The screen that asks me to allow a VPN connection doesn't react when I press okay, except for a grey text line that asks me to wait, it would activate Blokada. It does react when I press abort. I have exactly the same problem when I acitvete "watchdog", though it got checkmarked as activated. Since a reboot of the phone, the window pops up again and again for me to allow VPN connection for a while.

I have no installed VPN on my smartphone and a restart didn't help. Tried it with and without activated "watchdog".

Blokada is the version 3.7.022000, my phone is an LG G6 LG-H870DS, Android 8.0.0 

Best regards

Romi Klippert

P.S. congratulations, your app got advertised by the German tech magazine C'T. I saw it on Youtube.

### Steps to reproduce
1.
2.
3.

### Configuration
1. Blokada version

2. Device model

3. Android version / ROM

### Questions
1. How long have you been using Blokada? 

2. How long time ago has the problem started to happen? 

3.  How often does this problem happen to you?

**Comment by animeking9901** [comment]
If you're having trouble with the VPN allow prompt make sure no screen blue light filters or overlays are on

**Comment by Jaheira** [comment]
Thank you but there was no active filter. Also, the abort button works.

**Comment by rpeter85** [comment]
Do you have any other VPN service? Do you use DNS changer, for example as a Magisk module?

**Comment by Jaheira** [comment]
I found a solution, Blokada is active now! 
I deactivated two apps: MacroDroid and Round Screen Corners Pro, which turns the not-really-round corners of the LG G6 into really round corners by overlay. I cannot reproduce the mistake, I reactivated both apps an restarted my phone, but now I can activate Blokata every time.

Thank you to you all, I would never have done this experiment without your nudging because of blue light filters and your ongoing ideas to try out.
